tools/scenario_runner.py

```diff
@@ -1,40 +1,42 @@
 # -*- coding: utf8 -*-
```

Comment:
It's not 100% relative to your PR, but since you are touching this file can you:

1. Change `utf8` to `utf-8`
2. Add `#!/usr/bin/env python` since this seems to be an executable.